libs/wingsdk/src/platform/platform-manager.ts

```diff
@@ -74,33 +131,43 @@ export class PlatformManager {
       );
     }
 
-    let synthHooks: SynthHooks = {
-      preSynthesize: [],
-      postSynthesize: [],
-      validate: [],
-    };
-
-    let newInstanceOverrides: any[] = [];
+    return appCall!({
+      ...appProps,
+      _platformManager: this,
+      synthHooks: {
+        preSynthesize: this.hooks.preSynth,
+        postSynthesize: this.hooks.postSynth,
+        validate: this.hooks.validate,
+      },
```

Comment:
I really wanted to remove the responsibility for calling the Synth hooks from the App class that way we can enforce the call order contract at this level vs the App level, but Im going to save that for a future iteration.